plugins/bulk-action.yaml

```
@@ -21,11 +21,14 @@ spec:
   shortDescription: Do bulk actions on Kubernetes resources.
   caveats: |
     Usage:
-      kubectl bulk-action <get resourceTypes> (get|list|create|update|delete|remove|rollout) <command parameters>
+      kubectl bulk-action <get resourceTypes> (get|list|create|update|delete|remove|rollout|scale) <command parameters>
+
+    Help:
+      kubectl bulk-action help        
 
     This plugin needs the following programs:
     * grep
     * sed
   description: |
-    This plugin allows you to do bulk actions on Kubernetes resources.
+    This plugin allows you to do bulk actions on Kubernetes resources. Use carefully, its your own risk.
```

```suggestion
    This plugin allows you to do bulk actions on Kubernetes resources.
```
Isn't that obvious? Instead, you could say that your plugin will never ask for confirmations.